adam-core/src/main/scala/org/bdgenomics/adam/projections/ContigField.scala

```diff
@@ -17,18 +17,12 @@
  */
 package org.bdgenomics.adam.projections
 
-import org.bdgenomics.formats.avro.DatabaseVariantAnnotation
+import org.bdgenomics.formats.avro.Contig
```

Comment:
OOC, why does this show up as a file move? Any thoughts? May be just github being funky.